File: person.js
function Person(a, n) {
	Person.population += 1;
	this.age = a;
	this.first_name = n;
}

Person.population = 0;

var jon = new Person(23, "Jon");

jon.age;
jon.first_name;
jon.population;

Person.population;

jon.age = 50;
jon.age;

var Mary = new Person(35, "Mary");

Person.population;

typeof Person;

typeof jon;

File: person.rb
class Person
	@@population = 0
	def initialize(a, n)
		@age = a
		@first_name = n
		@@population += 1
	end

	def age
		@age
	end

	def first_name
		@first_name
	end

	def self.population
		@@population
	end
end

jon = Person.new(23, "Jon")

jon.age;	
jon.first_name;
jon.population		

Person.population	

jon.age = 50
jon.age

mary = Person.new 35, "Mary" 

Person.population
Person.class

jon.class